ast/src/program.rs

```diff
@@ -28,15 +28,25 @@ use std::fmt;
 /// Stores the Leo program abstract syntax tree.
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub struct Program {
+    /// The name of the program.
+    /// Empty after parsing.
     pub name: String,
+    /// Expected function inputs.
```

Comment:
Should we say 'main function inputs', i.e. the inputs for the main function. (At least for the time being, as this may get generalized as we evolve the transaction model.)